patches.xen/0001-xen-blkfront-add-partition_scan-module-parameter.patch

```diff
@@ -0,0 +1,54 @@
+From ea3c839d676d12be9f8d234bc14e14036fe33247 Mon Sep 17 00:00:00 2001
+From: Rusty Bird <rustybird@openmailbox.org>
+Date: Mon, 27 Jun 2016 18:25:14 +0000
+Subject: [PATCH] xen-blkfront: add partition_scan module parameter
+
+It is sometimes very useful to attach block devices from lesser to more
+trusted domains, but great care has to be taken not to expose parsers
+to unauthenticated, potentially harmful input.
```

Comment:
Can you provide such a use case, which do not involve mounting that device a moment later?